src/vm/gcenv.ee.cpp

```diff
@@ -13,8 +13,17 @@
 
 #include "common.h"
 
-#include "gcenv.h"
+#ifdef FEATURE_STANDALONE_GC
+#include "gcenv.ee.h"
+#else
+#include "../gc/env/gcenv.ee.h"
+#endif // FEATURE_STANDALONE_GC
+
+#ifdef FEATURE_COMINTEROP
+#include <windows.ui.xaml.h>
```

Comment:
I'll just get rid of the changes to this file - none of them are necessary anymore.